python-package/pyproject.toml

```diff
@@ -80,7 +80,7 @@ sdist.reproducible = true
 wheel.py-api = "py3"
 experimental = false
 strict-config = false
-minimum-version = "0.9.3"
+minimum-version = "build-system.requires"
```

Comment:
Another case where `scikit-build-core` is reducing the need for us to keep the same version in sync across 2 fields.

Setting `scikit-build-core>=0.10.1` is equivalent to setting this to `minimum-version = "0.10.1"` 😁